plugin-common/src/main/java/org/opennms/plugins/cloud/grpc/CloudLogService.java

```diff
@@ -0,0 +1,47 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.plugins.cloud.grpc;
+
+import io.grpc.MethodDescriptor;
+import io.grpc.Status.Code;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Logs Requests and sends them in batches to the cloud.
+ */
+@Slf4j
+public class CloudLogService {
+
+    public void log(long startTime, long endTime, MethodDescriptor methodInvoked, Code returnCode) {
```

Comment:
I suggest to add at least one test about CloudLogService.log()
e.g. verify times of log() being called.

Reply:
good point! I added a test.